Makefile.am

```diff
@@ -585,6 +585,8 @@ janus_pp_rec_SOURCES = \
 	postprocessing/pp-h264.h \
 	postprocessing/pp-av1.c \
 	postprocessing/pp-av1.h \
+	postprocessing/pp-avformat.h \
+	postprocessing/pp-avformat.c \
```

Comment:
Just a small nit, can you invert the order of .c and .h file here to match the others in the list?